backend/cellguide/pipeline/ontology_tree/tree_builder.py

```diff
@@ -350,89 +350,92 @@ def get_ontology_tree_state_per_tissue(self) -> Dict[str, OntologyTreeState]:
         all_states_per_tissue = {}
         tissue_by_cell_type = []
         for i, tissue in enumerate(self.uberon_by_celltype):
-            if " (" not in tissue:
-                logger.info(
-                    "Getting ontology tree state for tissue %s (%s/%s)", tissue, i + 1, len(self.uberon_by_celltype)
-                )
+            if not tissue.startswith("UBERON:") or " (" in tissue:
+                logger.info(f"Skipping tissue {tissue} because it is not a UBERON term.")
+                continue
 
-                tissueId = tissue
-                tissue_term = self.uberon_ontology[tissueId]
-                tissue_label = tissue_term.name
-
-                end_nodes = self.uberon_by_celltype[tissue]
-                uberon_ancestors = [i.id for i in tissue_term.superclasses()]
-
-                # filter out hemaotoietic cell types from non-whitelisted tissues
-                uberon_ancestors_in_whitelist = list(set(TISSUES_IMMUNE_CELL_WHITELIST).intersection(uberon_ancestors))
-                if len(uberon_ancestors_in_whitelist) == 0:
-                    end_nodes_that_are_not_hematopoietic = [
-                        e
-                        for e in end_nodes
-                        if HEMATOPOIETIC_CELL_TYPE_ID not in [i.id for i in self.ontology[e].superclasses()]
-                    ]
-                    if len(end_nodes_that_are_not_hematopoietic) == 0:
-                        logger.info(f"Not filtering out immune cell for {tissue_label}")
-                    else:
-                        end_nodes = end_nodes_that_are_not_hematopoietic
-                else:
-                    logger.info(f"Not filtering out immune cell for {tissue_label}")
+            logger.info(
```

Comment:
No diff here besides untabbing this block of code

Reply:
i'm surprised it passed our linter as it was before :o

Reply:
I untabbed it because I got rid of the outer if statement